cmd/testing/baseTLS.go

```diff
@@ -0,0 +1,66 @@
+package cmdtesting
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/bacalhau-project/bacalhau/pkg/bidstrategy/semantic"
+	"github.com/bacalhau-project/bacalhau/pkg/publicapi/client"
+	clientv2 "github.com/bacalhau-project/bacalhau/pkg/publicapi/client/v2"
+
+	"github.com/bacalhau-project/bacalhau/cmd/util"
+	"github.com/bacalhau-project/bacalhau/pkg/devstack"
+	noop_executor "github.com/bacalhau-project/bacalhau/pkg/executor/noop"
+	"github.com/bacalhau-project/bacalhau/pkg/logger"
+	"github.com/bacalhau-project/bacalhau/pkg/node"
+	"github.com/bacalhau-project/bacalhau/pkg/test/teststack"
+)
+
+type BaseTLSSuite struct {
+	BaseSuite
+}
+
+// before each test
+func (s *BaseTLSSuite) SetupTest() {
+	logger.ConfigureTestLogging(s.T())
+	util.Fatal = util.FakeFatalErrorHandler
+
+	computeConfig, err := node.NewComputeConfigWith(node.ComputeConfigParams{
+		JobSelectionPolicy: node.JobSelectionPolicy{
+			Locality: semantic.Anywhere,
+		},
+	})
+	s.Require().NoError(err)
+	ctx := context.Background()
+	requesterConfig, err := node.NewRequesterConfigWith(
+		node.RequesterConfigParams{
+			HousekeepingBackgroundTaskInterval: 1 * time.Second,
+		},
+	)
+	s.Require().NoError(err)
+
+	const serverCertPath = "../../testdata/certs/dev-server.crt"
+	const serverKeyPath = "../../testdata/certs/dev-server.key"
```

Comment:
Might it be best to use absolute paths to pass to the WithSelfSignedCertificate option?